dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -1,10 +1,43 @@
 class RubricsController < ApplicationController
+  include Rails.application.routes.url_helpers
+
   def edit
     @rubric = Rubric.find_by(id: params[:id])
     @lesson = @rubric.lesson
   end
 
   def new
     @lesson = Lesson.find_by(id: params[:lessonId])
+    @rubric = Rubric.new
+  end
+
+  def create
```

Comment:
Can you add a test for `create` that checks if a Rubric and LearningGoals are created correctly?